awx_collection/plugins/module_utils/tower_api.py

```diff
@@ -414,14 +414,14 @@ def default_check_mode(self):
             except(Exception) as excinfo:
                 self.fail_json(changed=False, msg='Failed check mode: {0}'.format(excinfo))
 
-    def delete_if_needed(self, existing_item, handle_response=True, on_delete=None):
-        # This will exit from the module on its own unless handle_response is False.
-        # If handle_response is True and the method successfully deletes an item and on_delete param is defined,
```

Comment:
this removal of `handle_reponse` seems fine to me since no modules are still using it.